### ISSUE TITLE
How to determine the number of sample training channels

### ISSUE THREAD

**Issue by ChenK19** [yzz673/Brant#3]
Hello, we are trying to continue the reconstruction task by mse based on your pre-trained model. I would like to ask how you construct the channel dimension of the training samples when constructing the training data set? For example, I have a patient with 80-channel deep electrode implantation. How to divide training samples from it as input?

Thanks for your any reply!

**Comment by DaozeZhang** [comment]
Thank you for your interest to our work! You only need to use all channels as training data, and there is no need to filter channels. The input shape of the model is `(bat_size, ch_num, seq_len, seg_len)`, for example `(bat_size, 80, seq_len, seg_len)` if there are 80 channels.

**Comment by ChenK19** [comment]
Thanks for your reply!
I tried loading the pre-trained model and performing inference (selecting 10 channels as the input sample dimension) and found that the loss was much greater than when using a model initialized with random weights. Does this mean it hasn't generalized well to my samples? May I need to do some preprocess like channel z-score or there is a common mean/deviation value like ImageNet needed to be minus/divided ?

**Comment by DaozeZhang** [comment]
Data should be normalized or standardized to keep it within a unified range. Moreover, we recommend that you finetune on your data based on your specific task, instead of directly proceeding to inference. After all, direct inference would only yield representations, without any task-specific classification or prediction head.

**Comment by ChenK19** [comment]
Sorry for my inaccurate wording. I was continuing the training on a reconstruction task with our data, and when I referred to the first loss printout during training as inference, that was a mistake. We noticed that the first loss printout with a pre-trained model is tens of times higher than that of a model initialized from scratch, hence the question. My sample input is (batch_size, 10, 150, 2000), where 2000 corresponds to 5 seconds at 400Hz. I wonder if the issue could be due to the seq_len being too large?
